### PR TITLE
Ini PR

### PR DESCRIPTION
**Pull request by kawaii-ghost**
Tinggalkan komentar

### CODE REVIEW

**Comment by aldy505** [comment]
@kawaii-ghost CI failed. Jalanin `pnpm run format` dulu.

**Comment by aldy505** [comment]
@elianiva can you help on executing `pnpm run format` please?

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the overall
+  community
```
Comment (elianiva) [inline .github/CODE_OF_CONDUCT.md:27]:
```suggestion
-   Demonstrating empathy and kindness toward other people
-   Being respectful of differing opinions, viewpoints, and experiences
-   Giving and gracefully accepting constructive feedback
-   Accepting responsibility and apologizing to those affected by our mistakes,
    and learning from the experience
-   Focusing on what is best not just for us as individuals, but for the overall
    community
```

File: .github/CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or advances of
+  any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email address,
+  without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
```
Comment (elianiva) [inline .github/CODE_OF_CONDUCT.md:38]:
```suggestion
-   The use of sexualized language or imagery, and sexual attention or advances of
    any kind
-   Trolling, insulting or derogatory comments, and personal or political attacks
-   Public or private harassment
-   Publishing others' private information, such as a physical or email address,
    without their explicit permission
-   Other conduct which could reasonably be considered inappropriate in a
    professional setting
```

File: .github/CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
+
+# Contributor Covenant Code of Conduct
```
Comment (elianiva) [inline .github/CODE_OF_CONDUCT.md:2]:
```suggestion
# Contributor Covenant Code of Conduct
```

**Comment by elianiva** [comment]
anu, ga bisa push langsung karena branchnya di repo @kawaii-ghost
but dw i have a solution

**Review by elianiva — CHANGES_REQUESTED** [review]
langsung add suggestion to batch aja

**Comment by herlangga72** [comment]
Biasa pada lupa prettier. Apa masukin ke build process aja??

**Review by elianiva — APPROVED** [review]

**Comment by elianiva** [comment]
thanks

**Pull request merged**